apps/api/src/app/portfolio/portfolio.controller.ts

```diff
@@ -230,6 +231,58 @@ export class PortfolioController {
     return { investments };
   }
 
+  @Get('dividends')
```

Comment:
Can you move this method before `getInvestments()`? These are sorted alphabetically.